src/python/s2_common.i

```diff
@@ -597,3 +723,72 @@ USE_COMPARISON_FOR_LT_AND_GT(S1Angle)
 USE_EQUALS_FN_FOR_EQ_AND_NE(S2Loop)
 USE_EQUALS_FN_FOR_EQ_AND_NE(S2Polygon)
 USE_EQUALS_FN_FOR_EQ_AND_NE(S2Polyline)
+
```

Comment:
Can this section be a separate PR? Everything else looks good, but now I have to go learn about repr()